qcodes/instrument_drivers/QDev/QDac.py

```diff
@@ -30,7 +31,8 @@ class QDac(VisaInstrument):
 
     # set nonzero value (seconds) to accept older status when reading settings
     max_status_age = 1
-
+    # this driver will be deprecated in favor of QDac_channels.py
+    @deprecate
```

Comment:
allow me to suggest to have a dedicated `deprecate` decorator for `__init__` - the warning message could then say that "some CLASS is deprecated, and other class should be used" instead of say "function SMTH is deprecated...", see here:
https://github.com/QCoDeS/Qcodes/blob/0f9176495d3bd02e6a05a221ef0b9635810c0f02/qcodes/utils/deprecate.py#L10-L10